### DISCUSSION TITLE
Is the complexity of reporting a typo preventing people from doing so ?

### DISCUSSION THREAD

**Discussion by Jaiket** [mdn/mdn-community#465] [Content] [answered]
Hello all, and thanks for helping me learn to mark up valid html/css.

I found a typo in an MDN document and a link at the bottom of the page which intends to allow me to correct it. The process feels extremely complicated. It appears to me that I must have GitHub and MDN accounts, understand search, forks, triaging.... Its tempting to say "that is way more effort than I am prepared to make to report a typo."

I would have liked a link named "report a typo in this document" which informed the relevant parties of the typo via a form.

As for the typo itself, I will attempt to complete the process.

**Comment by yin1999** [comment, marked as answer]
You can report it by clicking the "Report the content issue" link in the footer of the page where you find the issue.

![image](https://github.com/mdn/mdn-community/assets/15844309/4e139b97-0885-4da0-a026-72c320b9b1a7)

> For reporting other issue which may relate to Browser compatibility data, platform, etc., or if you want to fix it by yourself, please check the [guidelines](https://developer.mozilla.org/en-US/docs/MDN/Community/Issues#guidelines_for_reporting_an_issue).

**Reply by Jaiket** [reply]
Thankyou. I made a mistake.
